frontend/app/templates/components/transportation/existing-conditions/existing-conditions-steps.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="ui pointing menu">
```

Comment:
should this menu be above the table like in public schools instead of above the map? for consistency? 

Sorry, can't figure out how to update by previous reivew so starting a new one....

Reply:
Yes, the menu should be consistent with the Public Schools design, being under the map.